### DISCUSSION TITLE
Diarization: Using timestamps from whisper as external_vad but the output timestamps differ from whispers timestamps?

### DISCUSSION THREAD

**Discussion by tonytonfisk2** [NVIDIA-NeMo/NeMo#7735] [Q&A]
Have anyone else stumbled accross this issue and have some insights as too why it happens? I thought the VAD was the one who segmented and created timestamps? Since I don't run the VAD anymore shouldn't the timestamps just be from whisper? Does titanet or the msdd do that as well?

Another question I have is that I cannot process long audio files, it dies on the clustering part. Is there anyone that has a solution to this? I'm thinking of splitting long audio files but it doesn't sound like a good idea...

**Comment by jqueguiner** [comment]
whisper timestamp are a mess leading to hallucinations and omissions dont use it. furthermore there are really bad with a high alignement error rate.

**Reply by tykim123** [reply]
if so, which model can we use instead of Whisper model?
